src/components/Attendance/StudentWithWeeks.js

```diff
@@ -31,12 +31,13 @@ export default class StudentWithWeeks extends React.Component{
     getIsHomeworkChecked = (id, student) => {
         if (this.props.allHistory[student][id].homework === 1) {
             return true;
-        } else {return false}
+        } else {return false;}
```

Comment:
`return (this.props.allHistory[student][id].homework === 1)` is enough.
 Comparison operator already returns a boolean.

Reply:
Thanks for Review and  Response @M3kH .